sentry-android-core/src/main/java/io/sentry/android/core/UserInteractionIntegration.java

```diff
@@ -55,8 +55,13 @@ private void startTracking(final @Nullable Window window, final @NotNull Context
 
       final SentryGestureListener gestureListener =
           new SentryGestureListener(
-              new WeakReference<>(window), hub, options, isAndroidXScrollViewAvailable);
-      window.setCallback(new SentryWindowCallback(delegate, context, gestureListener, options));
+              new WeakReference<>(window),
+              new WeakReference<>(activity),
```

Comment:
We can get the window thru the `activity.window` right, so we don't need to pass both?

Reply:
yup, you're right, can be just activity